src/app.js

```diff
@@ -17,6 +19,7 @@ program
   .command("generate <language>")
   .option("--no-beta", "Disable the beta extension")
   .option("-d, --destination <destination>", "Output directory")
+  .option("--method <methodName>")
```

Comment:
Is it worth providing a description for the methodName option?

Reply:
It's a little hack really to aid development. Think if I were building that feature to be used by others I'd simply add in additional actions (other than just generate).